frameworks/hdfs/src/main/dist/krb5.conf

```diff
@@ -6,8 +6,9 @@
 [realms]
   {{SECURITY_KERBEROS_REALM}} = {
     kdc = {{SECURITY_KERBEROS_KDC_HOSTNAME}}:{{SECURITY_KERBEROS_KDC_PORT}}
+    admin_server = {{SECURITY_KERBEROS_KDC_HOSTNAME}}:{{SECURITY_KERBEROS_KDC_PORT}}
   }
 
 [domain_realm]
-  .{{FRAMEWORK_HOST}} = LOCAL
-  {{FRAMEWORK_HOST}} = LOCAL
+  .{{FRAMEWORK_HOST}} = {{SECURITY_KERBEROS_REALM}}
```

Comment:
Is there a ticket to track this for Kafka or ZK too? Do we want to add those as part of this PR?